apps/challenges/serializers.py

```diff
@@ -102,11 +106,22 @@ class ZipChallengeSerializer(ChallengeSerializer):
     """
     Serializer used for creating challenge through zip file.
     """
+    def __init__(self, *args, **kwargs):
+        super(ZipChallengeSerializer, self).__init__(*args, **kwargs)
+
+        context = kwargs.get('context')
+        if context:
+            image = context.get('image')
+            evaluation_script = context.get('evaluation_script')
+
```

Comment:
If context is None, then variables `image` and `evaluation_script` will not be defined. Can you simply assign their value in the if condition.

Also I am still not clear why are we adding this ? Was this missed or some new functionality ?